lms/static/scripts/frontend_apps/components/ErrorDialog.js

```diff
@@ -1,14 +1,21 @@
 import { createElement } from 'preact';
+import { useRef } from 'preact/hooks';
 
+import Button from './Button';
 import ErrorDisplay from './ErrorDisplay';
 import Dialog from './Dialog';
 
 /**
  * @typedef ErrorDialogProps
+ * @prop {boolean} [busy] -
```

Comment:
This is actually not used yet, but could be if we replace BaseDialog.. I sorta brought it in with the other retry onRetry prop